pkg/cli/cmds/server.go

```diff
@@ -32,6 +32,11 @@ var (
 			Usage:  "(components) Enable rke2 default cloud controller manager's service controller",
 			EnvVar: "RKE2_ENABLE_SERVICELB",
 		},
+		&cli.BoolFlag{
+			Name:   "single-node-sqlite",
```

Comment:
when we receive `disable-etcd` without `--server` we return this error here

`FATA[0000] invalid flag use; --server is required with --disable-etcd`

Reply:
maybe we could set as `single-server-sqlite`

Reply:
> when we receive disable-etcd without --server we return this error here

Right, but what of we made that the combination to enable sqlite, instead of adding a new flag?

Reply:
It work very well, I updated my PR with this combination.

Reply:
I like this combination as well, nice reuse of existing flags.